source/renderer/app/api/api.js

```diff
@@ -1657,11 +1655,7 @@ export default class AdaApi {
           ? {
               // N+2 epoch
               epochNumber: nextEpochNumber ? nextEpochNumber + 1 : null,
-              epochStart: nextEpochStartTime
-                ? moment(nextEpochStartTime)
```

Comment:
@yakovkaravelov why did you remove this? This is start time of the N+2 epoch and not the N+1 (next) one?

Reply:
@nikolaglumac I didn't remove this.
This is calculated in another file.

Reply:
Not as far as I can see: https://github.com/input-output-hk/daedalus/pull/2165/files#diff-6aa9e91ae277a5d21b8e1bb0b03ffafbR1633

Reply:
@nikolaglumac https://github.com/input-output-hk/daedalus/pull/2165/files#diff-faefd5629e4c983973ee387b12021c59R483
It is calculated here because epochLength should be observable variable of networkStatusStore.

Reply:
@yakovkaravelov I do not agree - please see attached screenshot.
The "next" and the "future" epochs can NOT have the same epoch start time...
Future epoch starts after the next epoch ends. Which means it's start time is the start time of the next epoch PLUS a length of a single epoch.

![see](https://user-images.githubusercontent.com/376611/92375023-d4296c80-f100-11ea-8696-59a621bf37d4.png)